Library/ViewModels/ManagePledgeViewModelTests.swift

```diff
@@ -909,13 +909,13 @@ internal final class ManagePledgeViewModelTests: TestCase {
 
       self.scheduler.advance()
 
-      XCTAssertEqual([], self.dataLakeTrackingClient.events)
-      XCTAssertEqual([], self.segmentTrackingClient.events)
+      XCTAssertEqual(["Page Viewed"], self.dataLakeTrackingClient.events)
```

Comment:
Could you explain why this changed from [] to Page Viewed?

Reply:
Previously there was no event tracked on that view model up to the point where the fix button is tapped. But now that I added a page viewed event, the `segmentTrackingClient.events` isn't empty anymore at that point, so it has to be appended.